src/managed/Microsoft.DotNet.PlatformAbstractions/RuntimeEnvironment.cs

```diff
@@ -101,6 +103,8 @@ private static string GetRIDOS()
                     return OperatingSystem.ToLowerInvariant();
                 case Platform.Darwin:
                     return "osx";
+                case Platform.FreeBSD:
+                    return "FreeBSD";
```

Comment:
Is this officially what we want the RID to be?  It looks inconsistent with our existing RIDs.  The other platforms don't have capital letters - `win`, `linux`, `osx`, `ubuntu`, etc.

I'm not sure how we make decisions on what string we use for RIDs.  @ericstj - do you know?